README.md

```diff
@@ -5,6 +5,20 @@
 
 [SPHERE.IO](http://sphere.io) is the first Platform-as-a-Service solution for eCommerce.
 
+## Deprecation notice
+
+We gained a lot of experiences and learnings with this SDK for our SPHERE.IO platform.
+To be transparent, let's list some drawbacks
+- maintainability: The testability wasn't proper ensured.
+- faster development: Integration new endpoints required a lot of code changes.
```

Comment:
*"The integration of new endpoints" or "Integrating new endpoints".
Or we can be more generic and say "Extending the SDK with new or custom functionality required a lot of code changes (or rather it was simply impossible)."